libs/ledger-live-common/src/families/xrp/bridge.integration.test.ts

```diff
@@ -9,8 +9,8 @@ import {
   // NotEnoughBalanceBecauseDestinationNotCreated,
   InvalidAddressBecauseDestinationIsAlsoSource,
 } from "@ledgerhq/errors";
-import { fromTransactionRaw } from "./transaction";
-import type { Transaction } from "./types";
+import { fromTransactionRaw } from "@ledgerhq/coin-xrp/lib/transaction";
+import type { Transaction } from "@ledgerhq/coin-xrp/lib/types";
 import { addNotCreatedRippleMockAddress } from "./bridge/mock";
 // import { formatCurrencyUnit } from "../../currencies";
 const newAddress1 = "rZvBc5e2YR1A9otS3r9DyGh3NDP8XLLp4";
```

Comment:
The `dataset` object should be define inside `coin-xrp` package.

Reply:
Done 👍